docs/user/integration/dynatrace/README.md

```diff
@@ -259,3 +262,37 @@ For easier access from the Kyma dashboard, add links to new navigation under **D
     ```
 
 2. If your Secret has a different name or namespace, then download the file first and adjust the namespace and name accordingly in the 'dataSources' section of the file.
+
+## Use Dynatrace Dashboards
+
+You can view metrics in Dynatrace dashboards:
```

Comment:
Does this sentence really add value? I believe we can drop it - or merge it with the following sentence:

```suggestion
1. To view metrics in Dynatrace dashboards, import the file [Telemetry Module Status](./telemetry-resource-metrics.json) as a Dynatrace dashboard. For details, see [Importing Dashboards](https://docs.dynatrace.com/docs/analyze-explore-automate/dashboards-classic/dashboards/dashboard-json#import-dashboard).
```

Also, I believe this and the following instruction could be in a numbered list, like in most step instructions.